doc/model_parameter.md

```diff
@@ -10,16 +10,19 @@ SQLFlow connects a SQL engine (e.g., MySQL, Hive, or MaxCompute) and TensorFlow
 
 ```SQL
 SELECT * FROM boston.train
-TO TRAIN xgboost.gbtree
+TO TRAIN xgboost.xgbregressor
 WITH
     objective ="reg:squarederror",
     train.num_boost_round = 30,
     validation.select = "SELECT * FROM boston.val LIMIT 8"
-COLUMN crim, zn, indus, chas, nox, rm, age, dis, rad, tax, ptratio, b, lstat
 LABEL medv
 INTO sqlflow_models.my_xgb_regression_model;
 ```
 
+#### Model Types
+
+`XGBOOST.XGBREGRESSOR`, `XGBOOST.XGBCLASSIFIER`, `XGBOOST.XGBRFCLASSIFIER`, `XGBOOST.XGBRANKER`, `XGBOOST.GBTREE`, `XGBOOST.GBLINEAR`, `XGBOOST.DART`
```

Comment:
Also add types for binary classification?

Reply:
In the `sklearn` API, `XGBClassifier` defaults to binary classification.

Reply:
Seems current implement only use `XGBOOST.XGBCLASSIFIER` with `multi:softprob`, how to use `binary:logistic` with the new model types?

Reply:
@typhoonzero correct me if I am wrong. The softmax loss and logistic loss are equivalent when dealing with binary classification. https://stats.stackexchange.com/a/254071

Reply:
It's true. The difference is that if always use `multi:softprob` you always have to specify `num_classes` parameter. Meanwhile, I didn't test if https://github.com/dmlc/xgboost/issues/4853 occurs now.

Reply:
@typhoonzero @shendiaomo Thanks for helping out. I will change `XGBClassifier`'s `objective` default to `binary:logistic`.